main.rb

```diff
@@ -0,0 +1,53 @@
+require_relative './school/actions/actions'
+
+class App
+  def initialize()
+    @actions = Actions.new
+  end
+
+  def init
+    main while @actions.keep_going
+  end
+
+  def do_action(option)
+    case option
+    when '1'
+      @actions.list_books
+    when '2'
+      @actions.list_people
+    when '3'
+      @actions.create_person
+    when '4'
+      @actions.create_book
+    when '5'
+      @actions.create_rental
+    when '6'
+      @actions.list_rentals
```

Comment:
- [ ] After entering a person's id to list its rentals the app crashes with this error "    filtered_rentals = @rentals.filter { |rental| rental.person.id == id }".